crux-bench/src/crux/bench.clj

```diff
@@ -54,24 +78,27 @@
     (log/infof "finished bench-ns '%s'." bench-ns)
 
     (doseq [result @*!bench-results*]
-      (println (json/write-str result)))))
+      (println (json/write-str result)))
+    (format-and-post-results-to-slack @*!bench-results*)))
 
 (defmacro with-bench-ns [bench-ns & body]
   `(with-bench-ns* ~bench-ns (fn [] ~@body)))
 
 (def ^:dynamic *node*)
 
 (defn with-node* [f]
-  (with-open [embedded-kafka (ek/start-embedded-kafka
-                              {:crux.kafka.embedded/zookeeper-data-dir "dev-storage/zookeeper"
-                               :crux.kafka.embedded/kafka-log-dir "dev-storage/kafka-log"
-                               :crux.kafka.embedded/kafka-port 9092})
-              node (api/start-node {:crux.node/topology 'crux.kafka/topology
-                                    :crux.node/kv-store 'crux.kv.rocksdb/kv
-                                    :crux.kafka/bootstrap-servers "localhost:9092"
-                                    :crux.kv/db-dir "dev-storage/db-dir-1"
-                                    :crux.standalone/event-log-dir "dev-storage/eventlog-1"})]
-    (f node)))
+  (try (with-open [embedded-kafka (ek/start-embedded-kafka
```

Comment:
can this use the new `crux.fixtures/with-tmp-dir` instead?

Reply:
how would you recommend using this? should I include crux-test, or a checkout to the file?